vars/sync2Prod.groovy

```diff
@@ -70,6 +70,9 @@ def call(String DESTINATION, String SYNC_PMM_CLIENT) {
                             /srv/repo-copy/apt/ \
                             10.10.9.209:/www/repo.percona.com/htdocs/apt/
 
+                        # Update /srv/repo-copy/version
+                        date +%s > /srv/repo-copy/version
+
```

Comment:
it should be done before rsync as version file should be updated on repo.percona.com